extension/README.md

```diff
@@ -113,7 +113,11 @@ The file has the following structure:
     "registries": [
         {
             "name": "My Private Registry",
-            "registry": "https://my-private.registry"
+            "registry": "https://my-private.registry",
```

Comment:
I'm not sure how useful the ability to set a default channel for an extension will be though. It might not be worth the trouble to fix that edge case.